cvmfs/server/cvmfs_server_merge_stats.sh

```diff
@@ -0,0 +1,99 @@
+#
+# This file is part of the CernVM File System
+#
+# Implementation of the "cvmfs_server stats command"
+
+clean_up() {
+    echo "Cleaning up"
+    echo "  Removing temporary files"
+    rm -rvf /tmp/cvmfs_server_merge_stats/*
+}
+
+# merge publish_statistics table
+cvmfs_server_merge_table() {
+  local db_file_1=$1
+  local db_file_2=$2
+  local output_db=$3
+  local TMP_DIR=/tmp/cvmfs_server_merge_stats
+
+  mkdir -p $TMP_DIR
+  sqlite3 $db_file_1 "SELECT key, value from properties" > $TMP_DIR/properties_values_1
```

Comment:
We need to add a package dependency to the `cvmfs-server` package for the `sqlite3` utlity.